icu4c/source/common/loclikely.cpp

```diff
@@ -921,11 +924,12 @@ _uloc_minimizeSubtags(const char* localeID,
     const char* trailing = "";
     int32_t trailingLength = 0;
     int32_t trailingIndex = 0;
+    UBool successGetMax = FALSE;
 
     if(U_FAILURE(*err)) {
         goto error;
     }
-    else if (localeID == NULL) {
+    else if (localeID == nullptr) {
```

Comment:
Changing `NULL` to `nullptr` here would introduce an inconsistency in this file.

At the moment, `NULL` is used consistently throughout the entire file. Plain and simple.

Your proposed change would make this file instead use 20% `nullptr` and 80% `NULL`. I don't see how that would be an improvement and it has absolutely nothing to do with what else this PR is about.

If you want to change `NULL` to `nullptr`, I suggest that you instead do that all at once as a separate change. That'll result in files with consistent usage and an easier to follow change history.

Reply:
I have been asked to change all NULL to nullptr in one function because I touch one line of NULL and someone quote we should always use nullptr in the "new code". I can switch back but it seems now different reviewers has different standard about this.